pkg/agent/nodeportlocal/npl_agent_init.go

```diff
@@ -31,30 +34,110 @@ import (
 	coreinformers "k8s.io/client-go/informers/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
 )
 
 // Set resyncPeriod to 0 to disable resyncing.
 // UpdateFunc event handler will be called only when the object is actually updated.
 const resyncPeriod = 0 * time.Minute
 
+func addRulesForNPLPorts(portTable *portcache.PortTable, allNPLPorts []rules.PodNodePort) error {
+	for _, nplPort := range allNPLPorts {
+		portTable.AddUpdateEntry(nplPort.NodePort, nplPort.PodPort, nplPort.PodIP)
+	}
+
+	if err := portTable.PodPortRules.AddAllRules(allNPLPorts); err != nil {
+		return nil
```

Comment:
if err != nil, it ignores the error? why `addRulesForNPLPorts` returns an error then?

Reply:
Missed to return an error here. Fixed it. Thanks!